### PR TITLE
Update nf-syn23664726-s3.yaml

### PR DESCRIPTION
**Pull request by allaway**
Adding a Tower user to be able to run [nf-synindex](https://github.com/Sage-Bionetworks-Workflows/nf-synindex) on the bucket.

### CODE REVIEW

File: config/prod/nf-syn23664726-s3.yaml [hunk 1]
```diff
@@ -21,6 +21,8 @@ parameters:
     - 'arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/robert.allaway@sagebase.org'
     - 'arn:aws:iam::694704239947:root' ## Nebula Genomics access
     - 'arn:aws:iam::694704239947:user/giri' ## Nebula Genomics access
```
Comment (zaro0508) [inline config/prod/nf-syn23664726-s3.yaml:23]: should we remove one of these from the list?

**Review by zaro0508 — APPROVED** [review]

File: config/prod/nf-syn23664726-s3.yaml (hunk 1 above)
Comment (allaway) [reply config/prod/nf-syn23664726-s3.yaml:23]:
```suggestion
    - 'arn:aws:iam::694704239947:root' ## Nebula Genomics access
```

Comment (allaway) [reply config/prod/nf-syn23664726-s3.yaml]: Yes! Done.

**Comment by allaway** [comment]
I found that one of 3k files was missing, so I would prefer not to mess with permissions until they xfer the final file. :) Will reopen when ready.

**Pull request closed without merging**